scripts/add_field_for_stuck_registrations/add_field_to_stuck_registration_contacts.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+import csv
+import sys
+
+from ndoh_hub.utils import rapidpro
+
+
+def process(contact_id):
+    params = {"stuck_in_consent_state": "True"}
+    rapidpro.update_contact(contact_id, fields=params)
+
+
+async def process_csv(args):
```

Comment:
This can just be `def process_csv(args):`

Reply:
thanks for picking this up